docs/reference/http.rst

```diff
@@ -0,0 +1,218 @@
+.. _ref_reference_http_api:
+
+HTTP API
+========
+
+Using HTTP, you may check the health of your EdgeDB instance, check metrics on
+your instance, and make queries.
+
+Your instance's URL takes the form of ``http://<hostname>:<port>/``. For
+queries, you will append ``db/<database-name>/edgeql``.
+
+.. note::
+
+    Here's how to determine your development EdgeDB instance's HTTP server URL:
```

Comment:
@fantix @tailhook @elprans we should probably have a command for stuff like this. https://github.com/edgedb/edgedb-cli/issues/914